simpleimage/make_rootfs.sh

```diff
@@ -66,16 +88,19 @@ UNTAR="bsdtar -xpf"
 METHOD="download"
 
 case $DISTRO in
-	arch)
+	arch|arch-lcd)
```

Comment:
Bad idea to use the $DISTRO variable for lcd variant. Please find another way so $DISTRO is always the distro and not something else.